GoogleDataTransport.podspec

```diff
@@ -34,7 +34,7 @@ Shared library for iOS SDK data transport needs.
   s.tvos.frameworks = 'SystemConfiguration'
 
   header_search_paths = {
-    'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}/GoogleDataTransport/"'
+    'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}/"'
```

Comment:
Any particular reason for this change?

Reply:
To handle the repo-relative imports that enable GDT and GDTCCT to be in the same SPM target.